README.md

```diff
@@ -90,6 +88,34 @@ You can also insert a code snippet by typing the name of the snippet in your cod
 
 The COBOL Language Support extension also supports user snippets. Add your custom snippets to the `COBOL.json` file in your user snippets folder.
 
+### Smart Tab
+
+Configure the smart tab function in your `settings.json` file to set tab stops at specific columns in the editor window. The cursor stops at these columns when you use **Tab** and **Shift+Tab** to move forwards and backwards in the code. You can specify custom tab stops, and use regular expressions to set conditional tab settings for different sections and divisions of the code.
+
+To configure the smart tab feature, open your `settings.json` file and change the `"cobol-lsp-smart-tab":` parameter.
+
+**Tip:** You can open your `settings.json` file directly from the COBOL Language Support extension settings. Under **Cobol-lsp: Smart-tab**, click **Edit in settings.json**.
+
+- Specify `false` to use the default tab settings for Visual Studio Code (every 4 columns).
```

Comment:
The default tab-width is configurable via "Indent Using Spaces" and "Indent Using Tabs" commands. Four spaces is only the default.